### DISCUSSION TITLE
Repo Contributors

### DISCUSSION THREAD

**Discussion by areed1192** [areed1192/td-ameritrade-api#32] [General]
Hello everyone! I'm excited to reach out and inquire if there are any individuals interested in lending their expertise to contribute to this project. I'm seeking a team of collaborators who can actively participate in managing the repository, handling pull requests, making valuable contributions, and providing overall project management support. While I will certainly continue to offer my assistance, it is vital to ensure the project's ongoing availability for all users. Your involvement would be greatly appreciated!

**Comment by cmpunches** [comment]
I'd be potentially interested but I'm a little confused about the state of its roadmap currently.

Schwab has purchased TD Ameritrade and converted all the accounts over that would be using this, so, are we talking about porting this to consume an API that Schwab provides (if any, I see lots of marketing about an API-like product but its in that weird fintech markety way where it's not clear if that API will be available to customers or if you need a special account type that's typically reserved for organization customers like some other banks do -- BOFA, that's a call out).

I guess I would need some clarification on this because it's my understanding that these API endpoints will no longer be reachable after today, and if they were, the accounts that would use them are now Schwab accounts.

**Reply by bferrarese** [reply]
Looks like Schwab is putting together a dev portal as well. 

https://developer.tdameritrade.com/content/trader-api-schwab-integration-guide

I can join the wagon and help if you guys need.

**Reply by areed1192** [reply]
Yeah, I created a sandbox account like two months ago, but still haven't received access. Also, I was limited to only advisor endpoints. I'm hoping they start opening more general access or at least give a status update on where they are at in the process.

**Reply by cmpunches** [reply]
I'm on the phone with one of their traders now that came over from TDA.  He's got me on hold looking into it.  I will update this thread shortly with hopefully some actionable intel.

**Reply by cmpunches** [reply]
Update on this.

To /use/ the API you'll need to sign up at:

https://developer.schwab.com

It'll make you jump through some hoops to get registered and set up MFA.

Once I got in there was no API reference.  Work in progress.

The trader did say that:

  a) there is a more advanced team that will be in tomorrow I can talk to that will have better details.
  
  b) the API is new and they're still figuring out what they need to do in some regards.
  
  c) they are attempting to make it feature-identical when they are done.
  
Perhaps after creating an APP for the OAuth stuff there will be a link to an API.  What you know is now more than what I know.

**Reply by cmpunches** [reply]
After watching this develop some more and seeing that you have to provide a callback URL and all this other nonsense on Schwab's setup, I'm pulling out.  They don't seem to have any clue what an API is or how it's used, or any other clue about what they're doing.  I'll have to find a new trader that provides API access.
